docs/sources/enterprise/white-labeling.md

```diff
@@ -0,0 +1,54 @@
++++
+title = "White-labeling"
+description = "White-labeling"
+keywords = ["grafana", "white-labeling", "enterprise"]
+aliases = ["/docs/grafana/latest/enterprise/white-labeling/"]
+type = "docs"
+[menu.docs]
+name = "White-labeling"
+parent = "enterprise"
+weight = 5
++++
+
+# White labeling
+
+> Only available in Grafana Enterprise v6.6+. Read more about [Grafana Enterprise]({{< relref "../enterprise" >}}).
+
+{{< docs-imagebox img="/img/docs/v66/whitelabeling_1.png" max-width="800px" caption="White labeling example" >}}
+
+This release adds new white labeling options to the `grafana.ini` file (can also be set via ENV variables).
+
+You can change the following elements:
+
+- Login Background
+- Logos and Favorite icon
+- Footer and sidebar links
```

Comment:
Better to be explicit here
- Login Background
- Login Logo
- Side menu top logo
- Footer & Help menu links
 - Fav icon (shown in browser tab)